src/config.js

```diff
@@ -71,6 +71,29 @@ const gaTrackingId = () => {
   }
 };
 
+const gaExperiments = () => {
+  switch (ndlaEnvironment) {
+    case 'local':
+      return {};
+    case 'dev':
+    // test IDs
+      return {
+        GOOGLE_ACCOUNT_ID: "74405776",
```

Comment:
Dette er en test konto for å verifisere under utvikling. Dette må konfigureres opp for de forskjellige miljøene.